platforms/minigame/common/engine/cache-manager.js

```diff
@@ -183,10 +183,13 @@ var cacheManager = {
         caches.sort(function (a, b) {
             return a.lastTime - b.lastTime;
         });
-        caches.length = Math.floor(caches.length / 3);
-        if (caches.length === 0) {
-            cleaning = false;
-            return;
+        let clearCachCount = Math.floor(caches.length / 3);
+        // 大于3则清理1/3， 小于等于3则全部清理
+        if(clearCachCount > 3){
```

Comment:
You can use caches.length >= 12 instead of clearCacheCount > 3.

And below, you can use caches.length < 3 instead of clearCacheCount === 0.

This way, when caches.length < 3, you won't need to execute Math.floor(caches.length / 3).

可以使用 caches.length >= 12 代替 clearCacheCount > 3

下面可以使用  caches.length < 3 代替 clearCachCount === 0

这样可以使得当 caches.length < 3 时不用执行 Math.floor(caches.length / 3)

Reply:
change to : 
        if(caches.length < 3){
            console.warn("Due to caching large files in the game, there is insufficient storage space. Now starting forced cleaning.");
        }
        else{
            caches.length = Math.floor(caches.length / 3);
        }